contrib/fonttools/showttf.c

```diff
@@ -5860,8 +5855,9 @@ static int readcff(FILE *ttf,FILE *util, struct ttfinfo *info) {
 	    }
 	}
     }
-    if (fontnames)
-        free(fontnames);
+    free(strings);
+    free(dicts);
```

Comment:
These are string arrays. Looking at line 6428, this will free the array structure of `strings` but not the strings. (Line 5574 analogous.)

Reply:
See line 3957 for an example of freeing these. Could be worth pairing `readcfffontnames` with `freecfffontnames` (and so forth) containing that code rather than repeating it -- more modular that way.